packages/react-scripts/template/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "React App",
```

Comment:
Maybe we should make the `name` more verbose than the `short_name` to press the point.
Also, I'd like to see some documentation explaining where and how these are used, and that Chrome favors `short_name`.